src/components/cc-img/cc-img.js

```diff
@@ -65,14 +65,16 @@ export class CcImg extends LitElement {
   }
 
   render () {
+    const altValue = this.accessibleName ?? '';
     const isLoading = (this.src != null && !this._loaded && !this._error);
     const isSkeleton = (this.skeleton || isLoading);
-    const displayText = (this.src == null || this._error);
+    const displayAccessibleName = (this.src == null || this._error);
     return html`
-      <div class="wrapper ${classMap({ skeleton: isSkeleton, loaded: this._loaded, text: displayText })}">
-        <img src=${ifDefined(this.src ?? undefined)} @load=${this._onLoad} @error=${this._onError} alt="">
-        ${displayText ? html`
-          <div class="error-msg">${this.text}</div>
+      <div class="wrapper ${classMap({ skeleton: isSkeleton, loaded: this._loaded, accessibleName: displayAccessibleName })}">
```

Comment:
**🎯 suggestion:** Most of the time, CSS class names are snake-cased so I would suggest this instead:
```suggestion
      <div class="wrapper ${classMap({ skeleton: isSkeleton, loaded: this._loaded, 'accessible-name': displayAccessibleName })}">
```